suites/pre-release/dcf/4.preSignedURLTest.js

```diff
@@ -37,25 +49,35 @@ async function fetchDIDLists(I) {
     ).then((res) => new Gen3Response(res));
 
     const projectAccessList = httpResp.body.project_access;
-
+    
     // initialize dict of accessible DIDs
     let ok200files = {}; // eslint-disable-line prefer-const
     // initialize dict of blocked DIDs
     let unauthorized401files = {}; // eslint-disable-line prefer-const
 
-    // adding record DIDs to their corresponding ACL key
-    // ( I.cache.records is created in BeforeSuite() )
-    I.cache.records.forEach((record) => {
-      // console.log('ACLs for ' + record['did'] + ' - ' + record['acl']);
-      // Filtering accessible DIDs by checking if the record acl is in the project access list
-      const accessibleDid = record.acl.filter(
-        (acl) => projectAccessList.hasOwnProperty(acl) || record.acl === '*', // eslint-disable-line no-prototype-builtins
-      );
+    // assemble 200 authorized ids
+    const authorized200Project = ['*'];
+    for (const i in projectAccessList) { // eslint-disable-line guard-for-in
+      authorized200Project.push(i);
+    }
+    console.log(`Authorized project : ${JSON.stringify(authorized200Project)}`);
 
-      // Put DIDs urls and md5 hash into their respective lists (200 or 401)
-      const theFiles = accessibleDid.length > 0 ? ok200files : unauthorized401files;
-      theFiles[record.did] = { urls: record.urls, md5: record.md5 };
-    });
+    for (const project of authorized200Project) {
+      const record200Resp = await I.sendGetRequest(
```

Comment:
this is really the only thing you would need to keep, the call to indexd with the did, to make sure you got a 200